testsuite/gnat2goto/tests/loops/loops.adb

```diff
@@ -0,0 +1,47 @@
+procedure Loops is   
+   type cnt is new Integer range  42 .. 45;
+
+   subtype subint is Integer range 5 .. 12;
+
+   type weekday is (Mo, Tue, We, Th, Fr, Sa, Su);
+   
+   z : Integer := 0;
+   i1 : Integer; -- must not conflict with loop
+
+begin
+	
+   --  loop with no label (implicit target for 'exit')
+   for i in reverse 1 .. 3 loop
+      z := 1;
+      exit when i=2;
+   end loop;
+   
+   --  loop with label and unconditional 'exit'
+   myloop : 
+   for i in cnt'Range loop
+      if i = cnt'Last then
+	exit myloop;
```

Comment:
style: indentation; rationale: tests (just like code) will be read many times, so in GNAT we also try to make them as readable as possible.